src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -706,7 +706,7 @@ private void DoBind(EndPoint endPointSnapshot, Internals.SocketAddress socketAdd
             {
                 try
                 {
-                    if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"SRC:{LocalEndPoint} Interop.Winsock.bind returns errorCode:{errorCode}");
+                    if (NetEventSource.IsEnabled) NetEventSource.Info(this, $"SRC:{LocalEndPoint} Bind returns errorCode:{errorCode}");
```

Comment:
Question somewhat unrelated to the actual PR, but shouldn't we consider moving these verbose duplicate lines into a helper class?

```csharp
static class SocketTrace
{
    [MethodImpl(MethodImplOptions.AggressiveInlining)]
    public static void Info(object source, string s)
    {
        if (NetEventSource.IsEnabled) NetEventSource.Info(this, s);
    }
}
```

Reply:
Not if we want to keep using interpolated strings.